.github/workflows/deploy-snapshots.yaml

```diff
@@ -31,4 +31,4 @@ jobs:
       env:
         MAVEN_USER: ${{ secrets.MavenUser }}
         MAVEN_PASSWORD: ${{ secrets.MavenPassword }}
-      run: mvn -B deploy --file prj/pom.xml -DskipTests -s .github/maven/synoki.xml -DaltDeploymentRepository=synoki-nexus::default::https://nexus.synoki.io/repository/maven-snapshots/
+      run: mvn -B deploy --file prj/pom.xml -DskipTests -s .github/maven/synoki.xml
```

Comment:
Again, if we need settings file, let's name it `settings.xml` instead of `synoki.xml`